src/Symfony/Component/Translation/Translator.php

```diff
@@ -506,4 +522,19 @@ private function getConfigCacheFactory()
 
         return $this->configCacheFactory;
     }
+
+    private function getCatalogueInternal($locale = null)
```

Comment:
We could rename this back to `getCatalogue()` and make it private once we do no longer implement `TranslatorBagInterface` (in 3.0).